esx_service/vmdk_ops.py

```diff
@@ -75,7 +75,7 @@
 OBJ_TOOL_CMD = "/usr/lib/vmware/osfs/bin/objtool open -u "
 OSFS_MKDIR_CMD = "/usr/lib/vmware/osfs/bin/osfs-mkdir -n "
 MKFS_CMD = BIN_LOC + "/mkfs.ext4 -qF -L "
-VMDK_CREATE_CMD = "/sbin/vmkfstools -d thin -c "
+VMDK_CREATE_CMD = "/sbin/vmkfstools"
```

Comment:
nit: would be better to have `VMDK_CREATE_CMD = "/sbin/vmkfstools -c"` 
not  a big deal since the string needs to be formatted anyways, just makes it more symmetric with '-U' option